### PR TITLE
feat: ping-3865 onboarding screens

### PR DESCRIPTION
**Pull request by yusufputra**
# Helio PR Checklist

## Please review all of the following checks before making PR

- [ ] Jika menambahkan sebuah komponen, apakah sudah dipastikan tidak ada komponen yang redundant, baik secara UI maupun fungsional?
- [ ] Periksa apakah komponen yang dibuat reusable (contoh: membuat komponen View untuk membuat Divider antar section)
- [ ] Periksa untuk penggunaan state management yang baik dan benar (contoh: Context, useState), terutama penggunaan local state untuk optimistic UI (Pengoperan state dan data melalui props dari parent ke children dan sebaliknya)
- [ ] Pastikan komponen yang dibuat sudah sesuai dengan module yang ada terutama penamaan dan penempatan untuk meningkatkan readability dan maintainability.
- [ ] Implementasi fallback error untuk menangani kesalahan tak terduga, mencegah aplikasi mengalami crash
- [ ] Evaluasi kompleksitas kode dan perbaiki logika yang kompleks dengan memecahnya menjadi fungsi atau komponen yang lebih kecil dan lebih mudah dimengerti.
- [ ] Apakah sudah resolve semua komentar dari CodeRabbit🐇?
- [ ] Apakah semua unit tests sudah Pass/Success? (Tidak ada bypass)
- [ ] Periksa semua perubahan teks Bahasa Inggris approved oleh Bastian?
- [ ] Apakah sudah menggunakan normalized.dimen/ normalizeFontSize untuk semua perubahan style?
- [ ] Pastikan untuk mengikuti [guidelines pembuatan komponen Guidelines About Loading and Layout](https://docs.google.com/document/d/1GHbvEtFrQmEQXYBV9dLrPI654n2DbPwS1qjedajLSiw/edit)

### Author's comment

if you have any unchecked items, please explain the reason why.

### Summary by CodeRabbit

- Refactor: Updated the text content in the onboarding slideshow component of the Sign In screen. The changes aim to enhance user understanding and provide clearer instructions during the onboarding process.

### CODE REVIEW

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 41f9ef1da11f3e2b367258e2fe880f5a1b52e19d and f5d4d18587665dfc9c43f4ccc4380098117404f6 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/screens/SignInV2/elements/SlideShow.js (3)
</details>


<details>
<summary>Files ignored due to filter (5)</summary>

* src/assets/background/bg_onboarding_top.png
* src/assets/background/fg_onboarding_full_1.png
* src/assets/background/fg_onboarding_full_2.png
* src/assets/background/fg_onboarding_full_3.png
* src/assets/background/fg_onboarding_full_4.png

</details>





---

File: src/screens/SignInV2/elements/SlideShow.js [hunk 1]
```diff
@@ -35,19 +35,19 @@ const SlideShow = ({handleLogin, onContainerPress = () => {}}) => {
...
-          {'messages from your friends. Ask for advice,'}
+          {'accountable even when in Incognito Mode.'}
           {'\n'}
 
-          {'feedback, and more!'}
+          {'To increase your score, just invite followers,'}
           {'\n'}
 
-          {'As always, abuse can be blocked in two clicks.'}
+          {'create posts and avoid being blocked!'}
```
Comment (bastian-purrer) [inline src/screens/SignInV2/elements/SlideShow.js:50]: Image description   CodeRabbit

The new text provides a clear explanation of how users can increase their karma scores. It's more informative than the previous version and gives users actionable steps to improve their visibility within the community.
```diff
-      title: 'Receive Incognito Messages',
-       text: (
-         <Text>
-          {'Share your link on other platforms to receive'}
-           {'\n'}
- 
-          {'messages from your friends. Ask for advice,'}
-           {'\n'}
- 
-          {'feedback, and more!'}
-           {'\n'}
- 
-          {'As always, abuse can be blocked in two clicks.'}
-         </Text>
-       )
+      title: 'Better Karma, Higher Visibility',
+       text: (
+         <Text>
+          {'Karma scores help the community stay'}
+           {'\n'}
+  
+          {'accountable even when in Incognito Mode.'}
+           {'\n'}
+  
+          {'To increase your score, just invite followers,'}
+           {'\n'}
+  
+          {'create posts and avoid being blocked!'}
+         </Text>
+       )
```

File: src/screens/SignInV2/elements/SlideShow.js [hunk 2]
```diff
@@ -77,7 +74,7 @@ const SlideShow = ({handleLogin, onContainerPress = () => {}}) => {
           {'Chats, posts, comments, and communities:'}
           {'\n'}
 
-          {'Find all your conversations separated'}
+          {'Find all your conversations, separated'}
```
Comment (bastian-purrer) [inline src/screens/SignInV2/elements/SlideShow.js:77]: Image description   CodeRabbit

The change is minor and doesn't affect the overall meaning of the sentence. However, it improves readability by making the sentence more grammatically correct.
```diff
-          {'Find all your conversations separated'}
+          {'Find all your conversations, separated'}
```

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 41f9ef1da11f3e2b367258e2fe880f5a1b52e19d and f5d4d18587665dfc9c43f4ccc4380098117404f6 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/screens/SignInV2/elements/SlideShow.js (3)
</details>


<details>
<summary>Files ignored due to filter (5)</summary>

* src/assets/background/bg_onboarding_top.png
* src/assets/background/fg_onboarding_full_1.png
* src/assets/background/fg_onboarding_full_2.png
* src/assets/background/fg_onboarding_full_3.png
* src/assets/background/fg_onboarding_full_4.png

</details>







<details>
<summary>Review comments generated (3)</summary>

* Review: 3
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by asaadam — APPROVED** [review]

**Pull request merged**